### DISCUSSION TITLE
Section component as a child of data provider not allowed

### DISCUSSION THREAD

**Discussion by TheFunkyBiskit** [Budibase/budibase#8340] [Help] [answered]
I'm trying to organize my form a bit better and it doesn't seem to be possible to nest a section component within a data provider, or anywhere within the tree underneath one. It does appear to be possible in the [video embedded in this docs page though](https://docs.budibase.com/docs/designing-horizontal-forms), which is a little odd. Any ideas?

**Comment by melohagan** [comment, marked as answer]
Hey @TheFunkyBiskit 

Sections cannot *directly* be a child of **Data Provider**.

They can only be a child of:
 - a screen
 - a form
 - a container

It is worth noting however that, as shown in the video, a form and a container could be a child of a data provider, which may have a section in it.

**Reply by TheFunkyBiskit** [reply]
@melohagan, thank you for the reply.

Unfortunately I don't believe this is entirely accurate.
I just tried to nest a section like so: Screen > Data Provider > Form > Section, and I get the warning: "Section cannot be a child of Data Provider". Any ideas?

**Reply by melohagan** [reply]
Hey Funky, you need the container as well:

![Screenshot 2022-10-28 at 08 09 07](https://user-images.githubusercontent.com/101575380/198525385-2e810bc7-a528-48aa-8ffb-564a6891033b.png)

**Reply by TheFunkyBiskit** [reply]
That did the trick! Thank you!
